js/main.js

```diff
@@ -1,2 +1,125 @@
 $(function () {
+
+// Rock Paper Scissor routine input user choice, generate random computer response
+// find out who won, update score and then present result in table on screen
+// There seems to be a bias in the program toward the user -- cause not clear
+// First define globale variables that are used to display results or simplify flow:
+  let userWins=0
+  let machineWins=0
+  let machineInput=NaN
+  let userInput=NaN
```

Comment:
@AtleLohrmann The input corresponds to a string when it's assigned, so having either an empty string or `null` would be a better candidate for a default value. However, Great job on experimenting with other data types for default values.